code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -120,11 +120,11 @@
 	weight = 1
 	delay = 1 MINUTES // Prevents rule start while head is offstation.
 	cost = 10
-	requirements = list(101,101,70,40,30,20,20,20,20,20)
```

Comment:
requirement changes aren't needed also add monkestation edit comments

Reply:
Should be all set, now. Re-enabled roundstart revs, latejoins are disabled.